lib/helpers/getDomNodePath.js

```diff
@@ -1,32 +1,9 @@
-export function getDomNodePath(el){
-  if (!el.nodeName) return '';
+import uniqueSelector from 'unique-selector';
 
-  var stack = [];
-  while ( el.parentNode != null ) {
-    // console.log(el.nodeName);
-    var sibCount = 0;
-    var sibIndex = 0;
-    for ( var i = 0; i < el.parentNode.childNodes.length; i++ ) {
-      var sib = el.parentNode.childNodes[i];
-      if ( sib.nodeName == el.nodeName ) {
-        if ( sib === el ) {
-          sibIndex = sibCount;
-        }
-        sibCount++;
-      }
-    }
-    if ( el.hasAttribute('id') && el.id != '' ) {
-      stack.unshift(el.nodeName.toLowerCase() + '#' + el.id);
-    } else if ( sibCount > 1 ) {
-      stack.unshift(el.nodeName.toLowerCase() + ':eq(' + sibIndex + ')');
-    } else {
-      stack.unshift(el.nodeName.toLowerCase());
-    }
-    el = el.parentNode;
+export function getDomNodePath(el){
+  if (!( el.nodeName && el.parentNode )){
+    return null; 
   }
 
-  return stack.slice(1).join(' > ');
+  return uniqueSelector(el);
```

Comment:
Since there is basically no logic here and this is merely delegating to a library [that has it's own suite of tests](https://github.com/cypress-io/unique-selector/blob/master/test/unique-selector.js), I've opted not to write additional unit tests.